server.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const morgan = require('morgan');
+const mongoose = require('mongoose');
+const httpErrors = require('http-errors');
+const debug = require('debug')('authdemo:server');
+
+const handleError = require('./lib/handle-error');
+const parserBearerAuth = require('./lib/parse-bearer-auth');
+const authRouter = require('./route/auth-router');
+//TBD 
+
+const app = express();
+const port = process.env.PORT || 3000;
+const mongoURI = process.env.MONGODB_URI || 'mongodb://localhost/authdev';
+
+mongoose.connect(mongoURI);
+
+app.use(morgan('dev'));
+
+app.all('/', parserBearerAuth, function(req, res){
```

Comment:
u mean line 11? for authorization for the future but not determined hence i put TBD underneath - sorry should have been more articulate